modules/common/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/AtomicFieldsLengthValidator.scala

```diff
@@ -53,14 +52,13 @@ object AtomicFieldsLengthValidator {
   private def validateField(
     event: EnrichedEvent,
     atomicField: LimitedAtomicField
-  ): Either[ValidatorReport, Unit] = {
+  ): Either[AtomicFieldValidationError, Unit] = {
```

Comment:
```suggestion
  ): Either[FieldLengthError, Unit] = {
```
once we have 
```
sealed trait AtomicError
case class FieldLengthError(...) extends AtomicError
```
(other comment)